File: PayPal/src/main/java/com/braintreepayments/api/PayPalTokenizeCallback.java
package com.braintreepayments.api;

import androidx.annotation.NonNull;

/**
 * Callback for receiving result of
 * {@link PayPalClient#tokenize(PayPalPaymentAuthResult, PayPalTokenizeCallback)}.
 */
public interface PayPalTokenizeCallback {

    /**
     *
     * @param payPalResult a success, failure, or cancel result from the PayPal flow
     */
    void onPayPalResult(@NonNull PayPalResult payPalResult);
}

File: PayPal/src/main/java/com/braintreepayments/api/PayPalTokenizeCallback.kt
package com.braintreepayments.api

/**
 * Callback for receiving result of [PayPalClient.tokenize].
 */
fun interface PayPalTokenizeCallback {

    /**
     * @param payPalResult a success, failure, or cancel result from the PayPal flow
     */
    fun onPayPalResult(payPalResult: PayPalResult)
}
